### PR TITLE
Added method to add/unset a cookie from context

### PR DESCRIPTION
**Pull request by svenkubiak**
I have added a method to add and unset a cookie from the context.

### CODE REVIEW

File: ninja-core/src/main/java/ninja/Context.java [hunk 1]
```diff
@@ -668,4 +668,18 @@ private HTTP_STATUS(int code) {
      * @return true if the content type is to set {@code application/xml} 
      */
     boolean isRequestXml();
+
+    /**
+     * Adds a cookie to the servlet response
```
Comment (raphaelbauer) [inline ninja-core/src/main/java/ninja/Context.java]: without "servlet" please - Just "Adds a cookie to the response"

File: ninja-core/src/main/java/ninja/Context.java [hunk 2]
```diff
@@ -668,4 +668,18 @@ private HTTP_STATUS(int code) {
...
+     * 
+     * @param cookie Ninja cookie
+     */
+    void addCookie(Cookie cookie);
+
+    /**
+     * Removes a cookie from the servlet response
```
Comment (raphaelbauer) [inline ninja-core/src/main/java/ninja/Context.java]: without "servlet" please - Just "Removes a cookie to the response"

File: ninja-servlet/src/main/java/ninja/servlet/ContextImpl.java [hunk 3]
```diff
@@ -332,6 +331,17 @@ public boolean hasCookie(String cookieName) {
 
         return ninjaCookies;
     }
+    
+    @Override
+    public void addCookie(Cookie cookie) {
```
Comment (raphaelbauer) [inline ninja-servlet/src/main/java/ninja/servlet/ContextImpl.java:336]: We need a test for both addCookie and unsetCookie. Can be mocked. But we need it to make sure the feature does not get removed or crippled by accident :)

**Comment by raphaelbauer** [comment]
Very useful! I added some tiny remarks :)

**Comment by svenkubiak** [comment]
I have added the unit tests and updated the javadoc. Refactored a deprecated call in testGetRequestUri a long the way ;-)

**Comment by raphaelbauer** [comment]
Delicious!

**Pull request merged**